### ISSUE TITLE
jupyter labextension uninstall --all?

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/jupyterlab#6006]
Similar to `jupyter labextension update --all`, it would be useful to be able to uninstall all the extensions at once without recreating a conda environment.

**Comment by vidartf** [comment]
Good idea! The code for the update subcommand can serve as a decent base for what changes to do to the uninstall subcommand. Relevant place to start looking: https://github.com/jupyterlab/jupyterlab/blob/ae3bc129065705b96da79f2d16bef861f7a81c9b/jupyterlab/commands.py#L214-L238

**Comment by Madhu94** [comment]
I'm picking this up.

**Comment by Madhu94** [comment]
Opened https://github.com/jupyterlab/jupyterlab/pull/6058 for this
